packages/cluster-ui/src/sortedtable/sortedtable.module.scss

```diff
@@ -97,6 +97,10 @@
 
 }
 
+.break-line {
+    white-space:pre-wrap !important;
```

Comment:
It would be ideal if we can find a way around using an `!important` flag.

Reply:
I did try other option without using it, but none of them were working unfortunately. If you have any ideas, I'm open to it!